cosmovisor/README.md

```diff
@@ -66,6 +69,17 @@ Note that blockchain applications that wish to support upgrades may package up a
 
 The `DAEMON` specific code and operations (e.g. tendermint config, the application db, syncing blocks, etc.) all work as expected. The application binaries' directives such as command-line flags and environment variables also work as expected.
 
+
+### Detecting Updates
+
+`cosmovisor` is pooling the `$DAEMON_UPDATE_INFO_FILE` (default to `$DAEMON_HOME/data/upgrade-info.json`) file for new update instructions. The following heuristic is applied to detect update:
++ When, starting `cosmovisor` doesn't know much about currently running upgrade, except the binary (which is either in `current/bin/` or `genesis/bin` if the former doesn't exists). It tries to read the `current/upgrade_name.txt` file to get an information about the current update name.
++ If `current/upgrade_name.txt` doesn't exist then we assume that `upgrade-info.json` is an upgrade request. So, if the `upgrade-info.json` when starting a cosmovisor but `current/upgrade_name.txt` doesn't exist then we try to make an upgrade according to the `name` attribute in `upgrade-info.json`.
```

Comment:
As far as I understand, the use of `upgrade_name.txt` was replaced with `upgrade-info.json`. I think your latest commit is not reflected in the documentation. Can you help clarify this?

Reply:
```suggestion
+ If `cosmovisor/current/upgrade-info.json` doesn't exist then `cosmovisor` assumes that `data/upgrade-info.json` is an upgrade request. If `data/upgrade-info.json` when starting a cosmovisor but `cosmovisor/current/upgrade-info.json` doesn't exist, then `cosmovisor` tries to make an upgrade according to the `name` attribute in `data/upgrade-info.json`.
```